events/productionizing-jupyter-notebooks/README.md

```diff
@@ -0,0 +1,69 @@
+# Productionizing Jupyter Notebooks with Versatile Data Kit (VDK)
+
+## Table of Contents
+- [Goals](#Goals)
+- [Agenda](#Agenda)
+- [Purpose](#Purpose)
+- [Background](#Background)
+  * [Objective](#Objective)
+- [Tutorial](#Tutorial)
+- [Exercises](#Exercises)
+- [Lessons Learned](#Lessons-Learned)
+- [Feedback](#Feedback)
+- [Where to Find Us](#Where-to-Find-Us)
+
+## Presenter
+[Duygu Hasan](https://github.com/duyguHsnHsn) <a href='https://www.linkedin.com/in/duygu-hasan/'><img src="https://img.shields.io/badge/LinkedIn-0077B5"></a>
+
+## Goals
+- Understand the challenges associated with transitioning notebooks to production.
+- Introduce the VDK solutions to address these challenges.
+
+## Agenda
+- Discuss strategies for productionizing Jupyter Notebooks.
+- Showcase hands-on examples of these challenges in the Jupyter UI.
+
+## Purpose
+The purpose of this scenario is to demonstrate how to operationalize Jupyter notebooks using the Versatile Data Kit (VDK) Jupyter integration. By the end of this guide, you'll understand how to:
+* Create a data job with VDK within a Jupyter notebook.
+* Write a data workflow in a notebook and make it ready to be put in a production environment.
+
+## Background
+### Objective:
```

Comment:
Again those are not the objective. I feel we discuss this a few times. 

This is the setup of the tutorial. The tutorial walks you through standard ETL process  retrieve, clean, transform, load data and addresses the challenges often people in notebooks have while doing these things.